contents/node/nodejs.md

```diff
@@ -167,11 +169,11 @@ const Promise = require('bluebird'); // import a non-core module
 
 Node will first check if the module identifier passed to `require` is a core module or a relative path. If so, it will return the core module or the object which references the value of `module.exports` in the specified file path respectively. Otherwise, Node will attempt to load the module from the `node_modules` folder in the parent directory of the current module.
 
-## Benefit 8: Good support
+### Benefit 8: Good support
```

Comment:
I think a more descriptive title for this section would be `Large ecosystem` or `Large developer community`